Beta_Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -0,0 +1,1648 @@
+    agents_id = argToList(demisto.args().get('agents_ids'))
+    context = {}
+
+    agents_groups = move_agent_request(group_id, agents_id)
+
+    # Parse response into context & content entries
+    if agents_groups.get('agentsMoved') and int(agents_groups.get('agentsMoved')) > 0:
+        agents_moved = True
+    else:
+        agents_moved = False
+    date_time_utc = datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%SZ')
+    context_entries = contents = {
+        'Date': date_time_utc,
+        'AgentsMoved': agents_groups.get('agentsMoved'),
+        'AffectedAgents': agents_moved
+    }
+
+    context['SentinelOne.Agents(val.Date && val.Date === obj.Date)'] = context_entries
+
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['json'],
+        'Contents': contents,
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': tableToMarkdown('Sentinel One - Shutdown Agents \n' + 'Total of: ' + str(
+            agents_groups.get('AgentsMoved')) + ' agents were Shutdown successfully', contents, removeNull=True),
+        'EntryContext': context
+    })
+
+
+def get_agent_processes_request(agents_ids=None):
+
+    endpoint_url = 'agents/processes'
+
+    params = {
+        'ids': agents_ids
+    }
+
+    response = http_request('GET', endpoint_url, params)
+    if response.get('errors'):
+        return_error(response.get('errors'))
+    if 'data' in response:
+        return response.get('data')
+    return {}
+
+
+def get_agent_processes():
+    """
+    Retrieve running processes for a specific agent.
+    Note: This feature is obsolete and an empty array will always be returned
+    """
+    headers = ['ProcessName', 'StartTime', 'Pid', 'MemoryUsage', 'CpuUsage', 'ExecutablePath']
+    contents = []
+    context = {}
+    agents_ids = demisto.args().get('agents_ids')
+
+    processes = get_agent_processes_request(agents_ids)
+
+    if processes:
+        for process in processes:
+            contents.append({
+                'ProcessName': process.get('processName'),
+                'CpuUsage': process.get('cpuUsage'),
+                'MemoryUsage': process.get('memoryUsage'),
+                'StartTime': process.get('startTime'),
+                'ExecutablePath': process.get('executablePath'),
+                'Pid': process.get('pid')
+            })
+        context['SentinelOne.Agents(val.Pid && val.Pid === obj.Pid)'] = processes
+
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['json'],
+        'Contents': contents,
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': tableToMarkdown('Sentinel One Agent processes', contents, headers, removeNull=True),
+        'EntryContext': context
+    })
+
+
+def get_threats_command():
+    """
+    Gets a list of threats.
+    """
+    # Init main vars
+    contents = []
+    context = {}
+    context_entries = []
+    title = ''
```

Comment:
Seems like there are functions that use title and functions that don't, should go either way - either use title everywhere or static title everywhere.

Reply:
In this case if no result there will be no title at all